modules/ROOT/pages/dataweave-selectors.adoc

```diff
@@ -23,7 +23,7 @@ DataWeave function.
 | Descendants | `..keyName` | Array of values of any matching descendant keys
 | Dynamic | See <<dynamic_selector>>. | &nbsp;
 | Key-value pair | `.&keyName` | Object with the matching key
-| Index | `[<index>]` | Value of any type at selected array index
+| Index | `[<index>]` | Value of any type at selected array index. To start indexing the last item of the Array, use `-1` because `0` is reserved as the first element in the Array and `-0` does not exist.
```

Comment:
I'd simplify this as `Negative numbers will index from the end of the array onwards`. I also think we should say what happens using an index beyond the array size.

Reply:
revised based on the suggestion above